source/_docs/automation/action.markdown

```diff
@@ -33,7 +33,7 @@ automation 2:
     notification_service: notify.paulus_iphone
   action:
     # Actions are scripts so can also be a list of actions
-    - service: "{{ notification_service }}"
+    - service: notify.notify
```

Comment:
This change is incorrect. The variable is defined just above. This example is an example service to show the feature of using automation variables.